controllers/utils_test.go

```diff
@@ -18,121 +18,153 @@ package controllers
 
 import (
 	"errors"
+	"fmt"
+	"math/rand"
+	"time"
 
 	policiesv1 "github.com/kubewarden/kubewarden-controller/pkg/apis/policies/v1"
-	"github.com/onsi/gomega/types"
 
 	. "github.com/onsi/gomega" //nolint:revive
+	"github.com/onsi/gomega/types"
 
 	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
-	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	timeout      = 120 * time.Second
+	pollInterval = 250 * time.Millisecond
+)
+
 var (
-	someNamespace = corev1.Namespace{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "some-namespace",
-		},
-	}
 	templatePolicyServer = policiesv1.PolicyServer{
 		Spec: policiesv1.PolicyServerSpec{
-			Image:    "some-registry/some-policy-server:latest",
+			Image:    "ghcr.io/kubewarden/policy-server:latest",
```

Comment:
Given that now it gets instantiated, I wonder if we should instead use a known working tag, just in case that `:latest` doesn't end up ready or doesn't behave correctly.

Reply:
@viccuad you are right, what about we use an env variable as we did for the k3s testcontainer?

Reply:
that sounds like a good idea :).

Reply:
done in: https://github.com/kubewarden/kubewarden-controller/pull/576/commits/4efdcc9538326cd596a4bde2f5a7732a49325bc0